rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
I've tried your solution and it works only by using `named` in the rollup.config.

Using `default` does not work if we export more than just `default`
BTW if we use `named` I believe we do not even need to `Object.assign(glamorous, {ThemeProvider})`

My rollup knowledge is too limited tho